2016/democracy.html

```diff
@@ -0,0 +1,169 @@
+---
+layout: default
+title: Democracy
+slug: democracy
+permalink: /democracy/
+---
+
+<header class="section page-topper">
+  <h1>Democracy</h1>
+  <p>We help people in 12 countries to provide citizens with sites that keep an eye on their parliaments. Meanwhile, here in the UK, we run <strong>TheyWorkForYou</strong> and <strong>WriteToThem</strong>.</p>
+  <p>But this year we’ve also been working on a massive initiative: the <strong>EveryPolitician</strong> project collects and shares data on politicians around the world so that anyone, anywhere, can use it for their own parliamentary monitoring site… or any other project.</p>
+</header>
+
+<div class="section section--white">
+  <h1>We started the year with a democracy win for Costa Rica…</h1>
+  <p>In January, YourNextRepresentative site <a href="https://www.mysociety.org/2016/01/29/crowd-sourcing-candidate-data-in-costa-rica-tusrepresentanteslocales/">TusRepresentantesLocales</a> launched in Costa Rica. It’s evolving into a permanent parliamentary monitoring project, QuinteRepresenta, which will cover the country’s mayors and councillors.</p>
+  <!-- https://www.flickr.com/photos/ingmar/5244331998/in/album-72157625553765114/ -->
+</div>
+
+<div class="section section--flipped section--democracy-brexit">
+  <h1>…And then things in the UK went a little crazy</h1>
+  <p>In the rollercoaster of a week that the Brexit results were announced and David Cameron resigned, <strong>110,265 people visited TheyWorkForYou</strong> and <strong>44,990 accessed WriteToThem</strong>. That’s up to six times the usual traffic. Phew!</p>
+  <p>Suddenly everyone was into politics. Fortunately, <a href="https://www.mysociety.org/2016/07/01/keeping-up-with-fast-moving-politics/">we were there for them</a>.</p>
+  <p>We also added a <a href="https://www.mysociety.org/2016/01/25/you-asked-for-it-new-voting-lines-on-theyworkforyou/">whole swathe of new voting topics</a> to TheyWorkForYou, including an Environment section, so it’s now even easier to see how any MP cast their vote on the issues that matter to you.</p>
+</div>
+
+<div class="section section--white section--democracy-stats">
+  <div class="row">
+
+    <div class="column-one-half">
+      <div class="stat">
+        <p class="stat__main">
+          <span class="stat__number">3.4 million</span>
+          <span class="stat__name">pieces of data</span>
+        </p>
+        <p class="stat__comment">
+          EveryPolitician now covers 71,934 politicians in 233 countries. Can you help us source data for <a href="http://everypolitician.org/needed.html">the last 12</a>?
+        </p>
+      </div>
+    </div>
+
+    <div class="column-one-half">
+      <div class="stat">
+        <p class="stat__main">
+          <span class="stat__number">91,342</span>
+          <span class="stat__name">UK parliamentary speeches</span>
```

Comment:
Matthew says: "150,331 2016 Commons/Lords/Westminster hall/public bill committees/NI Assembly/SP debates" - if we do it this way we'll have to drop the bit about politicians cramming more in, as we didn't include the other houses last year. I suppose we could put both figures?